day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,40 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(topping)
+    @toppings += ["#{topping}"]
+  end
+  
+  def remove_topping(topping)
+    @toppings -= ["#{topping}"]
+  end
```

Comment:
Interesting ruby trick here. Take a look at ruby array methods to see what built in methods might be more conventional to use.